auth/src/main/java/com/firebase/ui/auth/util/PlayServicesHelper.java

```diff
@@ -11,13 +11,9 @@
  * client application
  */
 public class PlayServicesHelper {
-    private static GoogleApiAvailability mApiAvailability;
-
+    @Deprecated
```

Comment:
Any reason for this change?  Why not preserve the availability object?

Reply:
Isn't it kinda pointless to store a static field in a static field? 😆 I would guess this is old code from before you could get the availability singleton like so: `GoogleApiAvailability.getInstance()`.

Is there any reason to store the singleton's reference in our class?

Reply:
I see, I forgot that the underlying object was a singleton as well.  :+1:

Reply:
Haha, cool beans.